mpxe/integration_tests/test_pca9539r.py

```diff
@@ -15,7 +15,7 @@ def test_pca9539r(self):
         time.sleep(0.1)
         self.pca9539r.sim.assert_store_values(self, False)
         value = True
-        for x in range(3):
+        for _x in range(3):
```

Comment:
I think the usual convention for iterating n times without using the iteration variable is to use `_` like `for _ in range(3):`